api/tests.py

```diff
@@ -1,3 +1,321 @@
 from django.test import TestCase
+from django.urls import reverse
+from django.contrib.gis.geos import Point
+from django.contrib.auth.models import User
 
-# Create your tests here.
+import json
+
+from infrastructure.models import ProjectStatus
+from infrastructure.factories import ProjectFactory, InitiativeFactory
+from locations.models import GeometryStore, PointGeometry
+from locations.factories import PointGeometryFactory, CountryFactory
+from facts.tests.organization_factories import OrganizationFactory
+from facts.tests.person_factories import PersonFactory
+
+
+class TestProjectStatusListView(TestCase):
+    def test_that_all_project_statuses_are_returned(self):
+        url = reverse('api:project-status-list')
+        response = self.client.get(url)
+        self.assertEqual(len(json.loads(response.content.decode())), len(ProjectStatus.STATUSES))
+
+
+class TestGeometryStoreDetailView(TestCase):
+
+    def setUp(self):
+        self.geometry_store = GeometryStore()
+        self.geometry_store.save()
+        point = PointGeometryFactory()
```

Comment:
To clarify, that's the default behavior when using the `DjangoModelFactory` which would be preferred when working with Django models. It doesn't look like that's the case for all of these.